src/templates/_image.tpl

```diff
@@ -0,0 +1,22 @@
+{{/*
+Builds a full OCI image reference from the given image object.
+
+Usage:
+  {{ include "papermc-server.image" (dict "image" .Values.container.image) }}
+
+If digest is "none", returns: <registry>/<name>:<tag>
+Otherwise returns: <registry>/<name>@<digest>
```

Comment:
Tag is required even when specifying digest.
At least, it works when having both the tag and digest defined.